bindings/jupyter-modules/jupyter-ma-causal/src/components/slidebar-svg/index.js

```diff
@@ -7,6 +7,10 @@ export default class Chart extends Component {
     width: PropTypes.number.isRequired,
     height: PropTypes.number.isRequired,
     extent: PropTypes.array.isRequired,
+    renderLeftLabel: PropTypes.bool,
+    renderRightLabel: PropTypes.bool,
+    leftLabel: PropTypes.string,
+    rightLabel: PropTypes.string,
```

Comment:
Yes, ideally we should do that, but because I the component render the full bar when the `x === null`, and render the maximum value for the label when `leftLabel === null`, so having additional variable for checking whether or not to render seems to be easier for now.